Update Future Enhancement (#343)
AY2324S2-CS2103-F08-2/tp: docs/DeveloperGuide.md

* Enhancing the graphical user interface to make it more user-friendly, i.e. more usage of the mouse as compared to the keyboard.
* Enabling localisation support: In future more options will be available for users to set their preferred phone number validation types (i.e. US Phone format), Extended Character Set for Naming validation (so that Chinese / Vietnamese / Tamil, etc names can be supported).
* More commands to improve the counsellor workflow: As with a typical application, our primary objective is to concentrate collection of patient and appointment data. As we progress, we plan to introduce updates and increase more commands to derive new insights from the data. We appreciate your understanding and patience as we work towards working with the users and making these advanced features available.

> * More valid check for user manual edit features in `json` files. Currently, adding non-legible value like `null` will cause the app to not launch. A future enhancement will include more valid check, allow skipping of non-legible data and noticing user of invalid data upon launching the app.

Reviewer: I think this one we might want to write in a more generalized way. Let me think of how we can do so

> * More valid check for user manual edit features in `json` files. Currently, adding non-legible value like `null` will cause the app to not launch. A future enhancement will include more valid check, allow skipping of non-legible data and noticing user of invalid data upon launching the app.

Reviewer 2: ```suggestion
* More validation checks for when user manually edits `json` files. Currently, adding non-legible value like `null` will cause the app to not launch. A future enhancement will include more validation checks, allow skipping of non-legible data and detecting invalid data upon launching the app.
```
